telemetry/vscode/src/generate.ts

```diff
@@ -406,10 +397,30 @@ function generateFile(telemetryJson: MetricDefinitionRoot, dest: string) {
         kind: StructureKind.TypeAlias,
     }
 
+    const span: InterfaceDeclarationStructure = {
+        kind: StructureKind.Interface,
```

Comment:
is there a way to add `docs` so the generated result will have a docstring ?

Reply:
Apparently its possible, I just added "'Represents a telemetry span for tracking and recording metric data.'"